packages/ui/src/Field.tsx

```diff
@@ -36,58 +30,46 @@ export interface FieldProps extends FieldWithLabelsProps {
     | "address"
     | "customSelect";
   rows?: number;
+  value?: any;
+  setValue?: any;
   options?: SelectListOptions;
   placeholder?: string;
   disabled?: boolean;
 }
 
 /**
- * Field is a fully uncontrolled component for creating various inputs. Fully uncontrolled means Field manages its own
- * state for the TextFields/Switches/etc, not the parent component. When values are updated, Field will pass the data to
- * the parent through the handleChange prop. You can set an initialValue, but you should not update initialValue
- * this prop with the result of handleChange.
- *
  * Note: You MUST set a key={} prop for this component, otherwise you may wind up with stale data. Just changing
  * initialValue will not work.
  *
  */
-export function Field(props: FieldProps) {
-  const [value, setValue] = useState(props.initialValue || "");
-  const [multiselectValue, setMultiselectValue] = useState(props.initialValue ?? []);
-
+export const Field = ({
+  name,
+  label,
+  labelColor,
+  height,
+  type,
+  rows,
+  value,
+  setValue,
+  options,
+  placeholder,
+  disabled,
+  errorMessage,
+  errorMessageColor,
+  helperText,
+  helperTextColor,
+}: FieldProps) => {
   // showCustomInput, customValue, and selectValue are all for type="customSelect"
   const [showCustomInput, setShowCustomInput] = useState(false);
   const [customValue, setCustomValue] = useState("");
-  const [selectValue, setSelectValue] = useState(props.initialValue || "");
-
-  const handleChange = (newValue: string) => {
-    if (props.type === "currency") {
-      newValue = newValue.replace("$", "");
```

Comment:
I think we'll still want this to handle currency/percent fields without the caller having to deal with this.

Reply:
Agreed! I'll get that back in here shortly.

Reply:
Good deal! Wasn't sure if it was permanent or temporary while you're working it out, just wanted to make sure!

Reply:
For sure, a good reminder! I was just making sure that as I make the updates within our primary app that the changes required are reasonable before polishing it up. I'll request a new review here once I'm good there. Thanks for the feedback!

Reply:
This should be there now. It's in a different spot (right near the text component) but should do the same thing!